pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -760,15 +760,44 @@ def setup_method(self, method):
         self.offset6 = BusinessHour(start='20:00', end='05:00')
         self.offset7 = BusinessHour(n=-2, start=dt_time(21, 30),
                                     end=dt_time(6, 30))
+        self.offset8 = BusinessHour(start=['09:00', '13:00'],
+                                    end=['12:00', '17:00'])
+        self.offset9 = BusinessHour(n=3, start=['09:00', '22:00'],
+                                    end=['13:00', '03:00'])
+        self.offset10 = BusinessHour(n=-1, start=['23:00', '13:00'],
+                                     end=['02:00', '17:00'])
 
     def test_constructor_errors(self):
         from datetime import time as dt_time
-        with pytest.raises(ValueError):
+        with pytest.raises(ValueError,
```

Comment:
Can you use `pytest.mark.parameterize` over `match`, `start` and `end`?

Reply:
Updated